alea/model.py

```diff
@@ -160,6 +161,8 @@ def generate_data(self, **kwargs) -> Union[dict, list]:
             This implementation won't allow you to call generate_data by positional arguments.
 
         """
+        if set(kwargs.keys()) - set(self.parameters.fittable):
+            raise ValueError("You cannot pass non-fittable parameters to generate_data")
```

Comment:
> |  | fittable | non-fittable |
> |---|---|---|
> | hypotheses | ✅ | 🤔 |
> | generate_values | ✅ | ❌ |
> | nominal_values | 🤔 | ✅ |

If you want to set non-fittable parameters, like `wimp_mass`,

```
    to_vary:
      {
        wimp_mass: [10, 50, 100],
      }
```

it will be auto deduced by [fuction](https://github.com/XENONnT/alea/blob/0c4bdc720847f736d45ac9bc4734b72b042b74ee/alea/submitter.py#L378) into `nominal_values`.

Reply:
Ah right, yes I think this sounds good! 😊

Reply:
Updated.